Rakefile

```diff
@@ -481,10 +481,13 @@ def check_parameter_file_format(tsvpath, n_deps, name)
   # For each line in file
   i = 1
   File.read(tsvpath, mode: "rb").each_line do |line|
+    next if line[0].start_with? "C"
```

Comment:
Won't we miss lines, e.g., when `Dependency=HVAC System Cooling Type` has a row starting with "Central"?